python/sglang/srt/server_args.py

```diff
@@ -1443,7 +1443,9 @@ def _handle_model_specific_adjustments(self):
         # TODO: currently, it is only supported in the single node scenario. https://github.com/flashinfer-ai/flashinfer/issues/2006
         # TODO: there is currently a bug on H20 device specifically, https://github.com/flashinfer-ai/flashinfer/issues/2204
         device_name = get_device_name()
```

Comment:
get_device_name() is defined in the link below, it returns `None` for CPU
https://github.com/sgl-project/sglang/blob/main/python/sglang/srt/utils/common.py#L1816